api/source/danger/append_peril.ts

```diff
@@ -107,8 +107,7 @@ export async function appendPerilContextToDSL(
       sandbox.danger.github.utils = recreateGitHubUtils(api)
     }
 
-    const anySandbox = sandbox as any
-    anySandbox.peril = peril
+    sandbox.peril = peril
```

Comment:
changes it this file are not critical - I should be able revert them without any problems if desired (to keep PR a bit leaner)